scripts/handle-user-data.service

```diff
@@ -0,0 +1,15 @@
+#  This file is part of Minishift
```

Comment:
Since we have to handle user data in a separate service file now then I don't think we need to have `rc.local` service as part of docker service file.

Reply:
Also disable the service?

Reply:
Also disable the service?

Reply:
@gbraad yes because we are not using any other script which is part of this service iirc.